index.js

```diff
@@ -136,11 +149,14 @@ class Bookend extends BookendInterface {
      * @param  {PipelineModel}  o.pipeline  Pipeline model for the build
      * @param  {JobModel}       o.job       Job model for the build
      * @param  {Object}         o.build     Build configuration for the build (before creation)
+     * @param  {String}         clusterName Cluster name
```

Comment:
same comment as above, brackets for optional param
```suggestion
     * @param  {String}         clusterName Cluster name
```